packages/cli/templates/webcomponents/igc-ts/projects/side-nav/files/src/app/app.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-return-assign */
+import {
+  defineComponents,
+  IgcNavDrawerComponent,
+  IgcNavDrawerItemComponent,
+  IgcNavDrawerHeaderItemComponent,
+  IgcRippleComponent,
+} from 'igniteui-webcomponents';
+import { Router } from '@vaadin/router';
+import { routes } from './app-routing.js';
+
+defineComponents(
+  IgcNavDrawerComponent,
+  IgcNavDrawerItemComponent,
+  IgcNavDrawerHeaderItemComponent,
+  IgcRippleComponent,
+);
+
+export class App extends HTMLElement {
+  connectedCallback() {
+    this.innerHTML = `
+      <div class="outer-wrapper">
```

Comment:
The outer wrapper doesn't seem to do much, not entirely sure it's needed

Reply:
Yeah. I removed it and changed some stylings in that file that affect charts and gauges,